src/featureSelectors.ts

```diff
@@ -60,6 +60,17 @@ export const makeGetHasEntityCreatedSuccess = ({ name }: { name: string }) => {
   );
 };
 
+export const makeGetIsSliceSuccessful = ({ name }: { name: string }) => {
```

Comment:
I'm not a huge fan of this selector because it means that if another status of HAS_*_SUCCESS is created, they have to remember to add that value here which is a potential bug. Instead, I think it's better to check if there is a status at all. We shouldn't actually care what it is. That means that this selector goes away and line 95 will check to see if the status is truthy.